src/components/header/HeaderComponent.tsx

```diff
@@ -11,6 +11,8 @@ import computeStyleSheet from './HeaderComponentStyles';
 export interface Props extends BaseProps {
   title: string;
   subTitle?: string;
+  carModel?: string;
+  carLicensePlate?: string;
```

Comment:
Try to be more generic, HeaderComponent is not used only for the Users.  
Maybe use something like: 
```suggestion
  subTitleSecondLine?: string;
 subTitleThirdLine?: string;
```